### DISCUSSION TITLE
如何导入快速输入

### DISCUSSION THREAD

**Discussion by tjw123hh** [fcitx5-android/fcitx5-android#443] [Q&A | 问答] [answered]
快速输入无法导入（导入错误：快速输入格式无效）

[test.txt](https://github.com/fcitx5-android/fcitx5-android/files/14383303/test.txt)
[test2.txt](https://github.com/fcitx5-android/fcitx5-android/files/14383314/test2.txt)

**Comment by rocka** [comment, marked as answer]
文件拓展名应为 `.mb`

**Reply by tjw123hh** [reply]
![Screenshot_2024-02-23-17-42-56-88](https://github.com/fcitx5-android/fcitx5-android/assets/56749271/0317201a-467d-4c1f-b412-ea4c8619d3a4)

**Reply by rocka** [reply]
https://github.com/fcitx5-android/fcitx5-android/commit/03b7f44289f3ee39d0d711c7d041ad17ae528275
升级到 0.0.8 或更高版本，不要用 Google Play 的版本。

**Reply by tjw123hh** [reply]
ok
